### PR TITLE
Show orphan annotations with a notice

### PR DESCRIPTION
**Pull request by tilgovi**

### CODE REVIEW

**Comment by tilgovi** [comment]
Still needed:

- [ ] Maybe some tests change, I haven't checked yet.
- [ ] Use the feature flag in `shouldShowThread`, test that better

**Comment by nickstenning** [comment]
screen shot 2015-09-21 at 17 55 43

**Comment by dwhly** [comment]
I'd remove the apology.  It's not necessarily our fault.  

"We cannot find the exact position of this annotation".

Also, doublechecking that this will be feature flagged.

**Comment by nickstenning** [comment]
> Also, doublechecking that this will be feature flagged.

:heavy_check_mark:

**Comment by seanh** [comment]
I've rebased this after a pretty nasty merge with master, hope I didn't break anything, tests are still passing and the diff against master looks the same as it did before (if I didn't miss anything)

**Comment by tilgovi** [comment]
Looks right to me at a glance.

**Comment by seanh** [comment]
It doesn't seem to be working though, I'm not seeing the footer on orphaned annotations :( Trying to figure it out..

**Comment by tilgovi** [comment]
Did you enable the feature? Are you seeing orphan annotations at all? Have you made a sufficiently large change to the content to actually cause an annotation to be orphaned?

**Comment by seanh** [comment]
Feature is enabled, I'm testing by annotating a page then completely replacing the contents of the page, I am seeing the orphaned annotation just without footer

File: h/templates/client/thread.html [hunk 1]
```diff
@@ -45,3 +45,7 @@
       ng-show="vm.shouldShowAsReply()">
   </li>
 </ul>
+
+<footer class="thread-anchor-notice" ng-if="vm.container.message.$orphan">
```
Comment (tilgovi) [inline h/templates/client/thread.html]: Here's the problem. This got messed up in some rebasing or refactoring. I may have pushed it with this problem to begin with.

The flag is `$anchored` not `$orphan`, and the meaning is inverted.

Comment (seanh) [inline h/templates/client/thread.html]: Yeah, I had found that but it seemed to be that way in your original commit, ok, that fixes it, thanks!

**Comment by seanh** [comment]
Alright, this looks good to me now, will merge when Travis passes

**Comment by judell** [comment]
@nickstenning I'm set up to test this on stage but since this didn't merge until after you updated stage earlier today I don't think I can observe the new behavior there.

**Comment by seanh** [comment]
Oops, https://github.com/hypothesis/h/issues/2521

**Comment by tilgovi** [comment]
Seems this feature was never turned on in production. Was that intentional?

**Comment by dwhly** [comment]
Yep. Prefer to have a real home for orphans.

**Pull request merged**